openreview/conference/matching.py

```diff
@@ -251,6 +251,92 @@ def _build_conflicts(self, submissions, user_profiles, get_profile_info):
             return self._build_profile_conflicts(ac_profiles, user_profiles)
         return self._build_note_conflicts(submissions, user_profiles, get_profile_info)
 
+    def append_note_conflicts(self, user_profile, build_conflicts=None):
+        '''
+        Create conflict edges between the given Notes and a single profile
+        '''
+
+        # Re-setup information that would have been initialized in setup()
+        submissions = list(openreview.tools.iterget_notes(
+            self.conference.client,
+            invitation=self.conference.get_blind_submission_id(),
+            details='original'))
+        get_profile_info = openreview.tools.get_neurips_profile_info if build_conflicts == 'neurips' else openreview.tools.get_profile_info
+        self.match_group = openreview.tools.replace_members_with_ids(self.client, self.match_group)
+        if not all(['~' in member for member in self.match_group.members]):
+            print(
+                'WARNING: not all reviewers have been converted to profile IDs.',
+                'Members without profiles will not have metadata created.')
+        
+        # Adapt single profile to multi-profile code
+        user_profiles = [user_profile]
+        user_profiles = _get_profiles(self.client, user_profiles, with_publications=build_conflicts)
```

Comment:
`_get_profiles` returns a dummy profile when the profile_id doesn't exist. I think we should throw an error if there is no profile found. 

We could compute the conflicts associated with emails instead of profile id but it can be very problematic for later. I would prefer to always compute the conflicts against existing OpenReview profiles.

Reply:
I was also thinking of incorporating a solution to this issue in this pull request since they seem related:
https://github.com/openreview/openreview-py/issues/482. If I were to detect:
1. No OpenReview profile detected
2. Only using a Gmail account

It would be at this step, right? If those are the two things that the issue is pointing to. Would throwing an exception if the user does not have an OpenReview profile be a reasonable solution to https://github.com/openreview/openreview-py/issues/482? I would also have to add that same check into the `_build_X_conflicts()` functions as well.